client/tomcatconf/commands.properties.in

```diff
@@ -485,6 +485,10 @@ updateVPCOffering=1
 deleteVPCOffering=1
 listVPCOfferings=15
 
+#### VPC OSPF config commands
+vpcOSPFConfig=15
+vpcOSPFConfigUpdate=1
```

Comment:
To enable the APIs by default, please use `authorized = {RoleType.Admin, ...}` in the @APICommand annotation for these API. Remove changes from this file, we should no longer be changing this file.